integration/testrunner/db.go

```diff
@@ -82,34 +99,85 @@ func makeBigQueryClient(ctx context.Context, tb testing.TB, projectID string) *b
 	return client
 }
 
-// This calls the database to check that an audit log exists. It uses the retries that are specified in the Config
-// file. This method assumes that only a single audit log will match, which constitutes success.
-func queryIfAuditLogExistsWithRetries(ctx context.Context, tb testing.TB, bqQuery *bigquery.Query, cfg *Config, testName string) {
+// This calls the database to validate that an audit log exists with expected format and validate how many logs we expect to match, in order to handle streaming use cases.
+// It uses the retries that are specified in the Config file.
+func validateAuditLogsWithRetries(ctx context.Context, tb testing.TB, bqQuery *bigquery.Query, cfg *Config, testName string, wantNum int, requireJustification bool) {
```

Comment:
Why do we need `testName`?

Reply:
I guess we don't need it. Qinhang used it when `return retry.RetryableError(fmt.Errorf("no matching audit log found in bigquery after timeout for %q", testName))`

Reply:
done